view/src/main/java/org/xcolab/view/pages/members/users/MembersController.java

```diff
@@ -179,8 +179,7 @@ public void downloadMembersList(HttpServletRequest request, HttpServletResponse
 
         if (membersPermissions.getCanDownloadMemberList()) {
             try (MemberListCsvWriter csvWriter = new MemberListCsvWriter(response)) {
-                List<Member> memberList = MembersClient.listMembers(null, null,
-                        null, null, true, 0, Integer.MAX_VALUE);
+                List<Member> memberList = MembersClient.listAllMembers();
```

Comment:
This change was not necessary, but I thought it might be nice to reduce code reuse

Reply:
Good thinking 👍